ui/app/components/policy-editor.js

```diff
@@ -63,7 +64,14 @@ export default class PolicyEditorComponent extends Component {
           this.policy.id
         );
       }
-    } catch (error) {
+    } catch (err) {
+      let error = 'Error saving policy';
```

Comment:
I reuse this pattern a few places - Is it worth adding a helper for or extending the existing helper?